app/theme/client/imports/components/messages.css

```diff
@@ -200,6 +200,9 @@
 			background-color: rgba(15, 34, 0, 0.05);
 
 			& .message-actions {
+				display: flex;	
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4943960)